packages/lib-react-components/src/Markdownz/Markdownz.js

```diff
@@ -51,7 +51,7 @@ class Markdownz extends React.Component {
   // Support image resizing, video, and audio using markdown's image syntax
   renderMedia (nodeProps) {
     let width, height
-    const imgSizeRegex = /=(\d+(%|px|em|rem|vw)?)x(\d+(%|px|em|rem|vh)?)/
+    const imgSizeRegex = /=(\d+(%|px|em|rem|vw)?)x(\d+(%|px|em|rem|vh)?)?/
```

Comment:
Use lazy matching here when matching digits, and explicitly specify nothing as an option for the units.
```suggestion
    const imgSizeRegex = /=(\d+?(|%|px|em|rem|vw))x(\d*?(|%|px|em|rem|vh))/
```

Reply:
This suggested `imgSizeRegex` does not work correctly in the case of `=100x100` in the alt tag of an image, the match captures and returns only `=100x` instead of both width and height.

Reply:
D’oh! I thought I’d copied this straight from the regex in the render method.

Reply:
I forgot to anchor the expression to the end of the alt text string. This should work, but isn't a required change at all.
```suggestion
    const imgSizeRegex = /=(\d+?)(|%|px|em|rem|vw)x(\d*?)(|%|px|em|rem|vh)$/
```

Try it out here, with tests:  https://regexr.com/5t7uq